src/StoredEvents/StoredEvent.php

```diff
@@ -150,7 +150,7 @@ protected static function getActualClassForEvent(string $class): string
         return Arr::get(config('event-sourcing.event_class_map', []), $class, $class);
     }
 
-    protected static function getEventClass(string $class): string
+    public static function getEventClass(string $class): string
```

Comment:
Unfortunately I had to make this method public, because I don't want to duplicate it's logic.